modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -371,19 +371,12 @@ class ApiClient:
             )
         elif isinstance(obj, (datetime.datetime, datetime.date)):
             return obj.isoformat()
-
         elif isinstance(obj, dict):
             obj_dict = obj
+        elif hasattr(obj, "model_dump") and callable(obj.model_dump):
```

Comment:
unfortunately the way to_dict/from_dict integrate this breaks in the face of the from/to dict layer the codegen still ads


as all from_dict call convert missing fields to None, pydantic has no chance to distinguish passed in from missing

Reply:
If both the serialization and deserialization logic has issues, then do they need to be solved at the same time? That is, if this merge request successfully improves upon serialization behavior, then can we consider improving deserialization as a separate problem?

If we do need to improve both serialization and deserialization logic at the same time, then do you have a suggestion as for how to do that?